### PR TITLE
feat: fix expo web

### PR DESCRIPTION
**Pull request by danilaplee**
this fixes the expo web implementation and adds a demo
Screenshot 2024-06-16 at 05 39 56

### CODE REVIEW

**Comment by steida** [comment]
Wow, thank you! It looks great!

File: packages/evolu-common-web/src/PlatformLive.ts
Comment (steida) [inline packages/evolu-common-web/src/PlatformLive.ts:1]: Why this change?

Comment (danilaplee) [reply packages/evolu-common-web/src/PlatformLive.ts:1]: Gonna remove this changes yeah

**Review by steida — APPROVED** [review]
LGTM

**Comment by danilaplee** [comment]
> LGTM

invited you to expo organization

**Comment by steida** [comment]
Isn't it possible to run it without email and password?

Screenshot 2024-06-18 at 22 57 32

**Comment by danilaplee** [comment]
> Isn't it possible to run it without email and password?
> 
> Screenshot 2024-06-18 at 22 57 32

yes it is possible if you connect the organization to the repo we can just automate it from there

**Comment by steida** [comment]
Imagine someone who wants to try Evolu with React Native. It should run with zero-config.

**Comment by danilaplee** [comment]
> > Isn't it possible to run it without email and password?
> > Screenshot 2024-06-18 at 22 57 32
> 
> yes it is possible if you connect the organization to the repo we can just automate it from there

actually the project to the repo

**Comment by steida** [comment]
How can I do that, please? :-)

**Comment by danilaplee** [comment]
> How can I do that, please? :-)

https://expo.dev/accounts/evolu/projects/evolu/github

**Comment by danilaplee** [comment]
> How can I do that, please? :-)

Screenshot 2024-06-18 at 23 02 38

**Comment by steida** [comment]
I have no idea what I should do 😇 I made you an Evolu admin; feel free to set it up. Thank you.

**Comment by steida** [comment]
What next?

Screenshot 2024-06-18 at 23 12 11
